build-system/tasks/issues-tasks.js

```diff
@@ -0,0 +1,255 @@
+var BBPromise = require('bluebird');
```

Comment:
Should have license

Reply:
Did most the changes based on your feedback. One things here @dvoytenko , what do you mean by having license? Thank you.

Reply:
done. Thank you.

Reply:
added the license